src/devices/bus/a2bus/uthernet.cpp

```diff
@@ -0,0 +1,77 @@
+// license:BSD-3-Clause
+// copyright-holders:R. Belmont
+/*********************************************************************
+
+    uthernet.cpp
+
+    Apple II Uthernet Card
+
+*********************************************************************/
+
+#include "emu.h"
+#include "emuopts.h"
```

Comment:
Is this actually necessary?  It’s just reading an option value into a data member that it never uses.